erpnext/patches.txt

```diff
@@ -515,7 +515,7 @@ erpnext.patches.v11_0.move_leave_approvers_from_employee #13-06-2018
 erpnext.patches.v11_0.update_department_lft_rgt
 erpnext.patches.v11_0.add_default_email_template_for_leave
 erpnext.patches.v11_0.set_default_email_template_in_hr #08-06-2018
-erpnext.patches.v11_0.uom_conversion_data #30-06-2018
+erpnext.patches.v11_0.uom_conversion_data #30-06-2018 #11-05-2020
```

Comment:
@Alchez We running this patch again

Reply:
Ah, you're right, had a brainfart.

However, I had a look at the patch, and it seems to be first deleting all the current records and then creating new ones. This is dangerous since any user-generated factors will get deleted.